tests/unit/test_gbq.py

```diff
@@ -82,6 +83,19 @@ def test__bqschema_to_nullsafe_dtypes(type_, expected):
         assert result == {"x": expected}
 
 
+@pytest.mark.parametrize(
+    ["query_or_table", "expected"],
+    [
+        ("SELECT 1", True),
+        ("dataset.table", False),
+        ("project-id.dataset.table", False),
```

Comment:
I would add some redundant whitespace around table ID's to make sure that doesn't confuse the function, i.e. to test that `strip()` is indeed applied.